examples/pub-sub-rewind/javascript/tailwind.config.js

```
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./index.html",
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
```

ESLint

[error] Replace `⏎····"./index.html",⏎····"./src/**/*.{js,jsx,ts,tsx}",⏎··` with `'./index.html',·'./src/**/*.{js,jsx,ts,tsx}'`

(prettier/prettier)